src/components/checkbox/index.tsx

```diff
@@ -82,6 +83,11 @@ export interface CheckboxProps extends TouchableOpacityProps {
    * Additional styling for checkbox and label container
    */
   containerStyle?: StyleProp<ViewStyle>;
+  /**
+   * Duration of check animation (in milliseconds)
+   */
+  animationDuration?: number;
```

Comment:
Hi @woen4 
I suggest adding a more generic prop - `animationOptions` and allow passing both `duration` and `easing`
WDYT?

Reply:
I would be even better, i will implement